assets/src/design-system/theme/helpers/outline.js

```diff
@@ -35,7 +35,7 @@ export const focusCSS = (accent, background) => css`
   outline: none;
   box-shadow: ${({ theme }) =>
     `0px 0px 0 2px ${background || theme.colors.bg.primary}, 0px 0px 0 4px ${
-      accent || theme.colors.border.focus
+      typeof accent === 'string' ? accent : theme.colors.border.focus
```

Comment:
When using `focusCSS` in a styled component, the first argument will be the `props` of the component. The function was written with the following type:
```ts
type (accent: string, background: string) => string;
```

but when using it the `accent` is generally an object. Adding a typecheck to make sure `accent` is a string makes it so that we don't try to style using an object.